<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPlant</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Meu Jardim</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>331</width>
          <height>371</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>401</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>469</width>
           <height>369</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>469</width>
           <height>309</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Adicionar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>289</width>
                <height>100</height>
               </size>
              </property>
              <property name="title">
               <string>Cenoura</string>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>10</y>
                 <width>91</width>
                 <height>91</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../Legumes/Cenoura.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>30</y>
                 <width>161</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Colheita em: 3 Dias</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>50</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Ultima regada: 18 Horas</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>70</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Nivel de Atenção: Baixo</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>289</width>
                <height>100</height>
               </size>
              </property>
              <property name="title">
               <string>Chuchu</string>
              </property>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>10</y>
                 <width>91</width>
                 <height>91</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../Legumes/Chuchu.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_9">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>30</y>
                 <width>161</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Colheita em: 1 Dias</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_10">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>50</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Ultima regada: 18 Horas</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_11">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>70</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Nivel de Atenção: Médio</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>289</width>
                <height>100</height>
               </size>
              </property>
              <property name="title">
               <string>Tomate</string>
              </property>
              <widget class="QLabel" name="label_12">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>71</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../Legumes/Tomate.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_13">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>30</y>
                 <width>161</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Colheita em: 7 Dias</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_14">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>50</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Ultima regada: 18 Horas</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_15">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>70</y>
                 <width>171</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Nivel de Atenção: Alto</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Cultivador</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Iluminação:</string>
        </property>
       </widget>
       <widget class="QLabel" name="iluminacaoLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Umidade:</string>
        </property>
       </widget>
       <widget class="QLabel" name="umidadeLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Temperatura:</string>
        </property>
       </widget>
       <widget class="QLabel" name="temperaturaLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="iluminacao_progressBar">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="umidade_progressBar">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="temperatura_progressBar">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QQuickWidget" name="quickWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>149</y>
          <width>321</width>
          <height>221</height>
         </rect>
        </property>
        <property name="resizeMode">
         <enum>QQuickWidget::SizeRootObjectToView</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="aquireButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>390</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Aquisitar</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Recomendações</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>331</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Localização:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>81</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>Rio de Janeiro - RJ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Estação:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Inverno</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Nível Jardim:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Alto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Médio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Baixo</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Baseado no seus dados, recomendamos:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>Chicórias.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Salsas</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Couve-Manteiga</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mostarda</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quiabos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rabanetes</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
